.pre-commit-config.yaml

```diff
@@ -33,6 +33,8 @@ repos:
         additional_dependencies:
           - types-requests
           - types-pyyaml
+          - pytest
+          - google-adk==0.5.0 # until https://github.com/mozilla-ai/any-agent/issues/287 is resolved
```

Comment:
@daavoo  TBH I don't understand why I needed to do this. But pre-commit was pulling in stubs for the wrong ver of google adk and throwing all sorts of type errors. How does precommit decide which version of packages to pull in?